connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTaskContext.java

```diff
@@ -109,4 +111,13 @@ public void resume(TopicPartition... partitions) {
     public Set<TopicPartition> pausedPartitions() {
         return pausedPartitions;
     }
+
+    @Override
+    public void disableOffsetCommit() {
```

Comment:
Are there any use cases where the user might want to disable commits only temporarily?